src/bookshelf/stest/src/bookshelf_bench.erl

```diff
@@ -80,6 +81,7 @@ init([]) ->
     %% basho_bench sets cwd to ./tests/current so we need to
     %% traverse up an extra directory to get to deps
     code:add_paths(filelib:wildcard("../../../deps/*/ebin")),
+    code:add_paths(filelib:wildcard("../../deps/*/ebin")),
```

Comment:
Anything using rebar3 doesn't actually use the `deps` folder anymore so it won't be there in bookshelf in a clean checkout, so I'm thinking '../../..' isn't applicable anymore.  A possible alternative is `../../../_build/default/lib/*/ebin`.